portality/view/admin.py

```diff
@@ -144,6 +147,83 @@ def article_endpoint(article_id):
     resp.mimetype = "application/json"
     return resp
 
+def _validate_authors(form, require=1):
```

Comment:
This should probably be a custom method on the FormContext, rather than defined in the base of the view